### PR TITLE
Added recipe for ua-parser

### PR DESCRIPTION
**Pull request by pmlandwehr**
Pinging @mattrobenolt in case they'd like to be added as a maintainer to the `ua-parser` builder on [conda-forge](http://conda-forge.github.io), a library of community-built [`conda`](http://conda.pydata.org) packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/ua-parser-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that's entirely fine too.

### CODE REVIEW

**Comment by mattrobenolt** [comment]
what

File: recipes/ua-parser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyyaml
```
Comment (jakirkham) [inline recipes/ua-parser/meta.yaml]: Is `pyyaml` suppose to be a run-time dependency too?

Comment (pmlandwehr) [inline recipes/ua-parser/meta.yaml]: In `setup.py`it's [listed under `setup_requires`]((https://github.com/ua-parser/uap-python/blob/master/setup.py#L186) so I don't believe so.

**Comment by pmlandwehr** [comment]
@mattrobenolt  [`conda`](http://conda.pydata.org) is a cross-platform (albeit python-centric) build tool heavily used in the scientific computing community. conda-forge is a community package repository. `ua-parser` is a dependency for [`psiturk`,](https://psiturk.org/) and no doubt for other scientific libraries it would be good to bring under the umbrella. If this isn't your thing it's no problem; I sent you the ping because you seem to have been the most active maintainer of `ua-parser`.

File: recipes/ua-parser/meta.yaml (hunk 1 above)
Comment (mattrobenolt) [inline recipes/ua-parser/meta.yaml]: It's not a runtime dependency. It's only needed to compile the yml file into a py file, but the distributed builds don't need it.

File: recipes/ua-parser/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{%set version = "0.7.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "e1f7cb16ccb68d6c93b19348e678500bec8cd1f692e49214dd38ca2d100ada9b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (mattrobenolt) [inline recipes/ua-parser/meta.yaml]: Can you not use the whl files?

File: recipes/ua-parser/meta.yaml (hunk 1 above)
Comment (mattrobenolt) [inline recipes/ua-parser/meta.yaml]: To be clear, this is only needed if you were building from source. So you shouldn't need it here at all.

**Comment by mattrobenolt** [comment]
@pmlandwehr ah, no thanks. :) I have too much stuff on my plate as it is. Feel free to ping me with questions, but I can't be a maintainer of this.

File: recipes/ua-parser/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyyaml
+
+  run:
+    - python
```
Comment (jakirkham) [inline recipes/ua-parser/meta.yaml]: Maybe we need `perl` too? Seems to have `perl` in its submodule.

File: recipes/ua-parser/meta.yaml (hunk 2 above)
Comment (mattrobenolt) [inline recipes/ua-parser/meta.yaml]: To be pedantic, this isn't "from source" either. The tarball is still a compiled asset. To build from source, you'd need to pull source from git, then run `setup.py install`. Without doing that, this is still a compiled distribution.

File: recipes/ua-parser/meta.yaml (hunk 3 above)
Comment (mattrobenolt) [inline recipes/ua-parser/meta.yaml]: You definitely do not need perl for this Python package.

Comment (jakirkham) [inline recipes/ua-parser/meta.yaml]: Ok, thanks for clarifying.

File: recipes/ua-parser/meta.yaml (hunk 2 above)
Comment (mattrobenolt) [inline recipes/ua-parser/meta.yaml]: Oh, never mind. I see you are doing that. :(

Comment (jakirkham) [inline recipes/ua-parser/meta.yaml]: Sure, we have had this discussion before. There are some people that want to compile from the repo directly and some that don't. We ended up letting this slide with PyPI and such.

File: recipes/ua-parser/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/ua-parser/meta.yaml]: Is it alright if we drop this then, @pmlandwehr?

Comment (pmlandwehr) [inline recipes/ua-parser/meta.yaml]: Removing `pyyaml` from the build requirements gets the standard omission error:
```bash
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.python.org/simple/pyyaml/r
Command failed: /bin/bash -x -e /opt/anaconda3/conda-bld/work/ua-parser-0.7.1/conda_build.sh
```

Comment (jakirkham) [inline recipes/ua-parser/meta.yaml]: Ack! 😱 Sorry, @pmlandwehr, must have missed that issue for some reason. Let's leave it alone after all.

**Comment by jakirkham** [comment]
LGTM

**Pull request merged**